cmd/otel-allocator/config/config.go

```diff
@@ -96,6 +99,12 @@ func unmarshal(cfg *Config, configFile string) error {
 	return nil
 }
 
+func createDefaultConfig() Config {
+	return Config{
+		CRScrapeInterval: DefaultCRScrapeInterval,
```

Comment:
Are there any other defaults that would be good to have here?

Reply:
i.e. would it be better for us to just embed something like `CommonPrometheusFields` in the config?

Reply:
Having had a quick look through that (https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#commonprometheusfields), I think there's way too many that wouldn't apply to TA, or which would apply, but we'd rather set them once at the Collector level and inherit. So I think we'll need to cherry pick the ones we want, unfortunately.

Reply:
Ah, you mean just for TA config, not for the spec in the CR? I guess we could do that. It'd still be a bit messy to have settable values that don't do anything, but if they're just exposed in the TA config and not the CR, it's less of a problem.

Reply:
holy crap there are a LOT of fields in there. I think it's okay to piece meal it on second thought.

Reply:
I can open an issue to decide what we want from there, exactly. Should that block this change, though?